src/fides/api/ops/service/privacy_request/request_runner_service.py

```diff
@@ -415,6 +421,22 @@ async def run_privacy_request(
                 _log_exception(exc, CONFIG.dev_mode)
                 return
 
+        if policy.get_rules_for_action(
+            action_type=ActionType.consent
+        ) and can_run_checkpoint(
+            request_checkpoint=CurrentStep.consent,
+            from_checkpoint=resume_step,
+        ):
+
+            await run_consent_request(
+                privacy_request=privacy_request,
+                policy=policy,
+                graph=dataset_graph,
```

Comment:
I'm currently passing in the same dataset graph under the assumption that we'll create saas ConnectionConfigs where the details to talk to the third party are expressed in the saas config, and that there's an attached DatasetConfig.

Current logic will skip all collections that aren't attached to a saas connector.  The saas-connector logic can then determine if the type of saas connector can manage a consent request.